docs/measurements/methods/basic.md

```diff
@@ -68,10 +68,10 @@ $$
 This section gives a brief overview over the K-Modulation method.
```

Comment:
since `methods` is now it's own tab, the methods could be split in different subpages

Reply:
It's already organized in that way, but the methods are very short, so I left it as is for now